ios_behave/conf/ios-properties.cfg

```diff
@@ -1,16 +1,23 @@
 [Driver]
 type: ios
-appium_app_strings: true
```

Comment:
I've never used this capability, but when launching the app is complaining because I think you need to pass the path for the localized strings , which in this case I don't think it's available